packages/devtools_app/lib/src/screens/deep_link_validation/validation_details_view.dart

```diff
@@ -0,0 +1,309 @@
+              ),
+              if (viewType == TableViewType.domainView || viewType == TableViewType.singleUrlView) ...[
+                const SizedBox(height: intermediateSpacing),
+                Text('Domain check', style: textTheme.titleSmall),
+                const SizedBox(height: denseSpacing),
+                _DomainCheckTable(
+                  controller: controller,
+                ),
+              ],
+              if (viewType == TableViewType.pathView || viewType == TableViewType.singleUrlView) ...[
+                const SizedBox(height: intermediateSpacing),
+                Text('Path check (coming soon)', style: textTheme.titleSmall),
+                _PathCheckTable(),
+              ],
+              const SizedBox(height: largeSpacing),
+              Align(
+                alignment: Alignment.bottomRight,
+                child: FilledButton(
+                  onPressed: () {
+                    controller.validateLinks();
+                  },
+                  child: const Text('Recheck all'),
+                ),
+              ),
+              if (viewType == TableViewType.domainView) ...[
+                Text('Associated deep link URL', style: textTheme.titleSmall),
+                Card(
+                  color: colorScheme.surface,
+                  shape: const RoundedRectangleBorder(),
+                  child: Padding(
+                    padding: const EdgeInsets.all(denseSpacing),
+                    child: Column(
+                      crossAxisAlignment: CrossAxisAlignment.start,
+                      children: linkData.associatedPath
+                          .map(
+                            (path) => Padding(
+                              padding: const EdgeInsets.symmetric(
+                                vertical: denseRowSpacing,
+                              ),
+                              child: Row(
+                                children: <Widget>[
+                                  if (linkData.domainErrors.isNotEmpty)
+                                    Icon(
+                                      Icons.error,
+                                      color: colorScheme.error,
+                                      size: defaultIconSize,
+                                    ),
+                                  const SizedBox(width: denseSpacing),
+                                  Text(path),
+                                ],
+                              ),
+                            ),
+                          )
+                          .toList(),
+                    ),
+                  ),
+                ),
+              ],
+            ],
+          ),
+        ),
+      ],
+    );
+  }
+}
+
+class _DomainCheckTable extends StatelessWidget {
+  const _DomainCheckTable({
+    required this.controller,
+  });
+
+  final DeepLinksController controller;
+
+  @override
+  Widget build(BuildContext context) {
```

Comment:
same comment here. This build method is very large, which makes it less readable